OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -501,7 +501,12 @@ Action InitInputPanel(Widget panel)
 					{ "ReplaySpeedSlowKey", "Slow speed" },
 					{ "ReplaySpeedRegularKey", "Regular speed" },
 					{ "ReplaySpeedFastKey", "Fast speed" },
-					{ "ReplaySpeedMaxKey", "Maximum speed" }
+					{ "ReplaySpeedMaxKey", "Maximum speed" },
+					{ "StatisticsBasicKey", "Statistics Basic" },
```

Comment:
Can we please change these to "Basic Statistics tab" and so on, so they read as proper english?